Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1351,41 +1351,32 @@ public function isMonolithicPhp()
             return false;
         }
 
-        $kinds = $this->findGivenKind(array(T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO, T_INLINE_HTML));
```

Comment:
Nice improvement!
Wouldn't be faster to iterate the tokens start at index 1 and if check for;
```php
if ($token->isGivenKind(array(T_INLINE_HTML, T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO))) 
```

the reason is as the PHPDoc of the method states, the first token must be an open tag, so if you find an open tag after index 0 the file is not monolithic